v2/googlecloud-to-googlecloud/src/main/java/com/google/cloud/teleport/v2/templates/TextIOToBigQuery.java

```diff
@@ -244,10 +242,9 @@ public interface Options
 
     void setOutputTable(String value);
 
-    @TemplateParameter.Text(
+    @TemplateParameter.GcsWriteFile(
```

Comment:
This should be a `GcsReadFile`.
GcsWriteFile allows selecting a non-existing file. GcsReadFile requires selecting an existing file. `javascriptTextTransformGcsPath` is actually an input parameter for the template and must exist.

Reply:
Fixed